AUDITING.md

```diff
@@ -0,0 +1,75 @@
+# Auditing
+
+This document defines how Platform Chaos audits chaos events. It lists what is tracked and how it is formatted.
+
+## Tracked Data
+
+When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the following properties are recorded and exported through a JSON object.
+  - An `id` unique to this event instantiation (`guid`). 
+  - A `chaos-type` of enumerable type `initiate`
+  - `user` and `system` that terminated the event
+  - `date` of event initiation (including time `UTC`)
+  - targeted `resources`. This is broken down by `subscriptionID` and then by `resourceGroup`
+  - Chaos `event` information (`eventID`, `eventName`, `eventURL`)
+
+Additionally an *Extension* may record additional information regarding the specificities of the event. This information is stored within an `extension` property.
+
+Every chaos event initiation/termination will be logged. The general structure of an event log is defined below:
+
+- _event log_ (Object)
+  - `id` (String) 
+    - GUID unique to this event instantiation
+  - `type` (String) 
+    - One of `'initiate'`, `'terminate'`
+  - `user` (String)
```

Comment:
is this the equivalent of the identity the access token was issued to? if so we should likely rethink the term `user` here, as that implies a person

Reply:
Yes it is equivalent to identity the access token was issued to. What would be a better attribute name? Should the value here be the access token or something else?